src/views/empty-state.html

```diff
@@ -2,13 +2,13 @@
   <div ng-if="$ctrl.config.icon" class="blank-slate-pf-icon">
     <span class="{{$ctrl.config.icon}}"></span>
   </div>
-  <h1 id="title">
+  <h4 id="empty-state-title-{{$id}}" class="h1 title">
     {{$ctrl.config.title}}
-  </h1>
-  <p id="info" ng-if="$ctrl.config.info">
+  </h4>
+  <p id="empty-state-info-{{$id}}" class="info" ng-if="$ctrl.config.info">
     {{$ctrl.config.info}}
   </p>
-  <p id="helpLink" ng-if="$ctrl.config.helpLink">
+  <p id="empty-state-helpLink-{{$id}}" class="helpLink" ng-if="$ctrl.config.helpLink">
```

Comment:
Same reply as above.  Just to confirm, I'm ok with the change if y'all are certain something like `empty-state-title` & `empty-state-helplink` isn't helpful for those who may need to tweak styles (though I'd prob shorten classes and go for `.empty-state .helplink` by preference, but I'll update to whatever is current convention).